libs/domain/user/address-form/src/address-form.component.ts

```diff
@@ -191,4 +212,29 @@ export class AddressFormComponent
       ${this.fieldRenderer.buildForm(form, this.formValues as FormValues)}
     </form>`;
   }
+
+  @query('form')
+  protected formX?: HTMLFormElement;
```

Comment:
Would be nice to have a better name than `formX` :)

Reply:
Yeah, but the whole component needs refactoring, so I prefer to leave it for later. We currently have a form property alrady.